src/test/java/com/chromediopside/service/ProfileServiceTest.java

```diff
@@ -0,0 +1,48 @@
+package com.chromediopside.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import com.chromediopside.model.GiTinderProfile;
+import com.chromediopside.model.Language;
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProfileServiceTest {
+
+  private static final String validAccessToken = System.getenv("TEST_ACCESS_TOKEN");
+  private static final String invalidAccessToken = "1nval1dt0k3n";
+  private static final String testLogin = System.getenv("TEST_LOGIN");
+  private static final String testAvatarUrl = System.getenv("TEST_AVATAR_URL");
+  private static final String testRepos = System.getenv("TEST_REPOS");
+  private Set<Language> testLanguagesList = new HashSet<>();
+
+  @Before
+  public void setup() throws Exception {
+    testLanguagesList.add(new Language("Java"));
+    testLanguagesList.add(new Language("HTML"));
+  }
+
+  @Test
+  public void validAccessToken() throws Exception {
+    ProfileService profileService = new ProfileService();
+    GiTinderProfile expectedProfile = new GiTinderProfile();
+    expectedProfile.setLogin(testLogin);
+    expectedProfile.setAvatarUrl(testAvatarUrl);
+    expectedProfile.setRepos(testRepos);
+    expectedProfile.setLanguagesList(testLanguagesList);
```

Comment:
Why don't you create a constructor for this purpose? That would be much easier to set up a new object.